tables_io/io_layer.py

```diff
@@ -358,3 +461,187 @@ def readPqToDataframes(basepath, keys=None, **kwargs):
         except Exception:  #pragma: no cover
             pass
     return dataframes
+
+
+def readPqToArrayDict(infile):
+    """ Open a parquet file and return a dictionary of `numpy.array`
+    """
+    df = readPqToDataframe(infile)
+    return dataframeToArrayDict(df)
+
+
+def readH5ToArrayDict(infile):
+    """ Open an h5 file and and return a dictionary of `numpy.array`
+    """
+    df = readHdf5ToDataframe(infile)
```

Comment:
Think I understood what you're doing here now - treating pandas h5 as different to regular hdf5?  That makes sense, but definitely worth documenting.

Reply:
Yep. good point.